fulp_modules/ipc/code/organs/stomach.dm

```diff
@@ -0,0 +1,22 @@
+/obj/item/organ/stomach/robot_ipc
+	name = "IPC micro cell"
+	icon = 'fulp_modules/ipc/icons/ipc_organs.dmi'
+	icon_state = "stomach-ipc"
+	w_class = WEIGHT_CLASS_NORMAL
```

Comment:
>micro cell
>WEIGHT_CLASS_NORMAL

bruh moment

Reply:
why is a human-sized creature running on a MICRO cell, anyway? shouldn't it be a MACRO cell or something?